src/content/app/tools/blast/components/blast-settings/BlastSettings.tsx

```diff
@@ -183,12 +186,15 @@ const BlastSettings = ({ config }: Props) => {
             <BlastJobSubmit />
           </div>
         </div>
-        <ButtonLink
-          className={styles.previousJobs}
-          to={urlFor.blastSubmissionsList()}
-        >
-          Jobs list
-        </ButtonLink>
+        <div className={styles.actionButtons}>
+          <ButtonLink
+            to={urlFor.blastUnviewedSubmissions()}
+            isDisabled={!unviewedBlastSubmissions.length}
+          >
+            Unviewed jobs
+          </ButtonLink>
+          <ButtonLink to={urlFor.blastSubmissionsList()}>Jobs list</ButtonLink>
```

Comment:
How many places are we using these two buttons in? Do you think it's time to combine them into a dedicated component for BLAST?